packages/plugin-vue/src/utils/error.ts

```diff
@@ -5,16 +5,22 @@ export function createRollupError(
   id: string,
   error: CompilerError | SyntaxError
 ): RollupError {
-  ;(error as RollupError).id = id
-  ;(error as RollupError).plugin = 'vue'
+  const { message, name, stack } = error
+  const rollupError: RollupError = {
+    id,
+    plugin: 'vue',
+    message,
+    name,
+    stack,
```

Comment:
Is this comma correct here? I think prettier would remove it :thinking:

Reply:
```suggestion
    stack
```